tests/models/test_boundary_condition.py

```diff
@@ -105,3 +105,204 @@ def test_boundary_condition_mixin(t_end: int):
         # Projecting the expected value to the subdomain.
         expected_val = bg.projection().T @ expected_val
         assert np.allclose(bc_val_prev_ts[bc_type.is_neu], expected_val[bc_type.is_neu])
+
+
+"""Here follows mixins related to testing of Robin limit cases, and eventually the test itself. """
+
+
+class Geometry:
```

Comment:
I think we can use `SquareDomainOrthogonalFractures` with the parameter `fracture_indices` equal to `[]`.